rust_src/src/frame.rs

```diff
@@ -0,0 +1,14 @@
+//! Generic frame functions.
```

Comment:
file should be `frames.rs` to be consistent.

Reply:
I forgot this one. I'll change this tomorrow